### ISSUE TITLE
EditorV2: tools/layers change should be recorded in history

### ISSUE THREAD

**Issue by TsFreddie** [teeworlds/teeworlds#2760]
When you switch to another layer, make some changes then undo, it reset your focus to the layer before you switched. This makes doing repeated minor adjustment really annoying.

**Comment by LordSk** [comment]
Yes this is planned already. I have not yet found a *great* solution for it though at the moment. For now it tracks what layer and group you had selected and restores them.
